src/compiler/crystal/tools/doc/method.cr

```diff
@@ -177,10 +177,22 @@ class Crystal::Doc::Method
     !@def.args.empty? || @def.block_arg || @def.yields
   end
 
+  def args_to_json
+    @def.args.map_with_index do |arg, i|
+      {
+        name:         arg.external_name,
+        type:         arg.type?.to_s,
+        default:      arg.default_value.to_s,
```

Comment:
Perhaps `nil`/`null` is clearer that there's no default, compared to empty string.